src/Protocol/Protocol_1_9.cpp

```diff
@@ -2665,8 +2665,11 @@ void cProtocol_1_9_0::HandlePacketUseEntity(cByteBuffer & a_ByteBuffer)
 	{
 		case 0:
 		{
-			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand)
-			m_Client->HandleUseEntity(EntityID, false);
+			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand);
+			if (Hand == 0)  // TODO: implement handle off-hand (1) actions
```

Comment:
I don't think that's such a good idea. The protocol should not depend on internal Cuberite representations. If we ever decide to change the enum for any reason, we'll suddenly and quietly break this protocol.